calculator.c

```diff
@@ -0,0 +1,77 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+#include<string.h>
+void removewhitespacesfromexpression(char input[]) {
+    int start = 0, start1 = 0;

```

Comment:
Can variable names match below standards, take care of this at all places: 

- Use descriptive names: Choose meaningful names that reflect the purpose of the variable.
- Use snake_case: Use lowercase letters with underscores to separate words.
ex : 
int total_count;
float average_score;
char student_name[50];


Global variables:
- Prefix with g_ or global_ to differentiate from local variables.
ex :
int g_total_count;

Constants:
- Use all uppercase letters with underscores to separate words.
ex:
#define MAX_BUFFER_SIZE 1024
const float PI = 3.14159